packages/plugin-core/src/workspace/tutorialInitializer.ts

```diff
@@ -60,35 +53,59 @@ export class TutorialInitializer
 
     const vpath = vault2Path({ vault: opts.vaults[0], wsRoot: opts.wsRoot });
 
-    fs.copySync(path.join(dendronWSTemplate.fsPath, "tutorial"), vpath);
-
+    // this is the currently active tutorial AB testing group.
     const ABUserGroup = MEETING_NOTE_TUTORIAL_TEST.getUserGroup(
       SegmentClient.instance().anonymousId
     );
 
-    const replaceString =
-      ABUserGroup === MeetingNoteTestGroups.show ? MeetingNoteTutorialText : "";
+    // this is the hierarchy of the treated tutorial.
+    // e.g. for the meeting note test group,
+    // the treated tutorial is in `dendron.tutorial.meeting-note.*`,
+    // so TREATMENT_NAME is `meeting-note`.
+    const TREATMENT_NAME = "meeting-note";
+
+    let filter: fs.CopyFilterSync;
+    let replacePattern: string;
+    switch (ABUserGroup) {
+      // swap out to appropriate enum case for current AB testing group
+      case MeetingNoteTestGroups.show: {
+        filter = (filePath) => {
+          if (path.extname(filePath) === ".md") {
+            const basename = path.basename(filePath);
+            return (
+              basename === "root.md" ||
```

Comment:
do we also need to copy over root.schema.yml?

Reply:
we do. this block is only concerned with paths that end with `.md`. all other paths (asset path and schema) are always copied over. (see line 79-81 just below)